src/pages/MainLayout.vue

```diff
@@ -1,5 +1,13 @@
 <template>
-  <div>
+  <div v-if="browserDetect">
+    <p> {{ $t('OUTDATED.FIRST_MESSAGE') }} <a
```

Comment:
You could use the [vue-i18n functional component](https://kazupon.github.io/vue-i18n/en/interpolation.html) to get slightly nicer translation strings, like here:

https://github.com/yunity/karrot-frontend/blob/61200a6eba32ca0daafd845961d183b79e350f7f/src/components/Settings/VerificationWarning.vue#L6-L18
with this message https://github.com/yunity/karrot-frontend/blob/61200a6eba32ca0daafd845961d183b79e350f7f/src/locales/locale-en.json#L333